document/src/vespa/document/base/field.cpp

```diff
@@ -91,15 +91,15 @@ Field::validateId(int newId) {
         throw vespalib::IllegalArgumentException(vespalib::make_string(
                     "Attempt to set the id of %s to %d failed, values from "
                     "100 to 127 are reserved for internal use",
-                    getName().c_str(), newId));
+                    vespalib::string(getName()).c_str(), newId));
     }
```

Comment:
Field:.getName() return ref to zero terminated string.

Reply:
Good catch, will remove wrapping for these